src/material/chips/chips.md

```diff
@@ -186,6 +186,8 @@ For both MatChipGrid and MatChipListbox, always apply an accessible label to the
 
 Always apply MatChipRemove to a `<button>` element, never a `<mat-icon>` element.
 
+When implementing the `removed` Output for MatChipRemove, ensure that removing chip is communicated to screen readers. One way to accomplish this is by sending a message with `LiveAnnouncer`. Otherwise, removing a chip will be only be communicated visually and some users might not be aware that it was removed.
+
```

Comment:
```suggestion
When using `MatChipRemove`, you should always communicate removals for assistive technology. One way to accomplish this is by sending a message with `LiveAnnouncer`. Otherwise, removing a chip may only be communicated visually.

```

Reply:
Accepted suggestion